next.config.js

```diff
@@ -182,12 +182,6 @@ const nextConfig = {
         destination: '/help',
         permanent: true,
       },
-      // Redirect legacy /discover page link to new /search page
-      {
-        source: '/discover',
-        destination: '/search',
-        permanent: true,
```

Comment:
@SudharakaP I just realized this was introduced as a 301 (permanent) redirect. So people who visited the page in the past may still be redirected. Just saying for us to be aware of this for future cases